### PR TITLE
feature to add approved labels after approved review

### PR DESCRIPTION
**Pull request by harshad16**
feature to add approved labels after approved review
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: aicoe/sesheta/actions/pull_request.py [hunk 1]
```diff
@@ -266,8 +291,8 @@
     return False
 
 
-async def needs_rebase(pull_request: dict = None) -> bool:
```
Comment (goern) [inline aicoe/sesheta/actions/pull_request.py]: why is this renamed?

File: aicoe/sesheta/actions/pull_request.py [hunk 2]
```diff
@@ -101,7 +126,7 @@
 
     _LOGGER.debug(f"checking if {pull_request['html_url']} needs a rebase label")
 
-    needs_rebase_actual = await needs_rebase(pull_request)
```
Comment (goern) [inline aicoe/sesheta/actions/pull_request.py]: s/// problem?

File: pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 120
-py36 = true
```
Comment (goern) [inline pyproject.toml]: uh nice, thx!

File: aicoe/sesheta/actions/pull_request.py (hunk 1 above)
Comment (harshad16) [reply aicoe/sesheta/actions/pull_request.py]: @goern as the function is checking if the PR is merged or mergeable. That's can be used for multiple things , why use it for only rebase. adding label would need a check if pr is merged or mergeable. so changed it to use it many other places without any issue.

File: aicoe/sesheta/actions/pull_request.py (hunk 2 above)
Comment (harshad16) [reply aicoe/sesheta/actions/pull_request.py]: what is the problem?

File: aicoe/sesheta/actions/pull_request.py (hunk 1 above)
Comment (goern) [reply aicoe/sesheta/actions/pull_request.py]: well, it specifically checks for the criteria of a merge, not for any label

Comment (harshad16) [reply aicoe/sesheta/actions/pull_request.py]: i understand it not checking for labels, however if a pr is merged there is not to added any label to it. so this function would be used other functions as well. 
Maybe it should be renamed to `check_merged` or `check_mergeable`!
